src/quicklist.c

```diff
@@ -854,7 +857,7 @@ REDIS_STATIC void _quicklistInsert(quicklist *quicklist, quicklistEntry *entry,
         full = 1;
     }
 
-    if (after && (entry->offset == node->count)) {
+    if (full && after && (entry->offset == node->count - 1)) {
```

Comment:
ping @oranagra, ```(entry->offset == node->count - 1)```
This is a minor error, and does not cause any side effects.
Like to #9113.